carbon-design-system/carbon
# fix(SideNav): pass additional props provided to SideNav to the rendered nav element

Closes #7903

pass additional props provided to `SideNav` to the rendered `nav` element.

#### Changelog

**New**

- SideNav pass additional props to rendered element

#### Testing / Reviewing

- Create `SideNav` element
- in it's props, pass an arbitrary prop, like `data-loating-menu-container={true}`
- verify that this attribute is rendered in the `nav` element
